common/asyncworkflow/queue/consumer/default_consumer.go

```diff
@@ -199,6 +200,9 @@ func (c *DefaultConsumer) processRequest(logger log.Logger, request *sqlblobs.As
 			ctx, cancel := context.WithTimeout(c.ctx, c.startWFTimeout)
 			defer cancel()
 			resp, err = c.frontendClient.StartWorkflowExecution(ctx, startWFReq, yarpcCallOpts...)
+			if isWorkflowAlreadyStartedError(err) {
```

Comment:
could we add a log? Or do we already have logs for this error?

Reply:
Good idea, done.